test/N3Writer-test.js

```diff
@@ -902,6 +903,4691 @@ describe('Writer', () => {
         writer.quadToString(new NamedNode('a'), new NamedNode('b'), new Quad(new NamedNode('a'), new NamedNode('b'), new NamedNode('c'), new NamedNode('g'))),
       ).toBe('<a> <b> <<<a> <b> <c> <g>>> .\n');
     });
+
+    testRelativizes('http://example.org/', [
```

Comment:
Cool cool cool. Thanks for the many, many tests.

How did you choose them?
Is there some kind of logic/structure to the test that comments could help capture?

Reply:
For every baseIRI, the tested nodes to relativize are the same. If you have the list of nodes, each IRI is being used once as baseIRI. Then the list of tested nodes also contains their "extended" variant, so just the IRI appended with "extended" to test that things like `http://ex.org/foo` don't match `http://ex.org.fooextended` in the way that it would relativize to `<extended>`, but properly relativize to `<fooextended>`.
I can try to summarize this in a comment above the tests.